CodenamesAI/src/Main.kt

```diff
@@ -13,16 +13,16 @@ fun main(args: Array<String>) {
 
     // Main game loop
     while (true) {
-        var currentHinter : HintAgent = if(isRedTeamTurn) redHinter else blueHinter
-        var currentGuesser : GuessAgent = if(isRedTeamTurn) redGuesser else blueGuesser
+        val currentHinter : HintAgent = if (isRedTeamTurn) redHinter else blueHinter
+        val currentGuesser : GuessAgent = if (isRedTeamTurn) redGuesser else blueGuesser
 
         // get hint
-        var hint = currentHinter.getHint(board, isRedTeamTurn)
+        val hint = currentHinter.getHint(board, isRedTeamTurn)
```

Comment:
`hint` gets reassigned later on in the loop - so I'm pretty sure this needs to remain "var"